LabelGenerator.py

```diff
@@ -88,11 +88,22 @@ def __init__(
     vertical_stride=24.3 * mm,
 )
 
+EJ_RANGE_24 = PaperConfig(
+    pagesize=A4,
+    sticker_width=63.5 * mm,
+    sticker_height=33.9 * mm,
+    sticker_corner_radius=2 * mm,
+    left_margin=6.5 * mm,
+    top_margin=13.2* mm,
+    horizontal_stride=66.45 * mm,
+    vertical_stride=33.9 * mm,
+)
+
 
 class StickerRect:
     def __init__(self, layout: PaperConfig, row: int, column: int):
         self.left = layout.left_margin + layout.horizontal_stride * column
-        self.bottom = layout.pagesize[1] - (layout.top_margin + layout.vertical_stride * row)
+        self.bottom = layout.pagesize[1] - (layout.top_margin + layout.vertical_stride * (row+1))
```

Comment:
I can't merge this as-is, as it breaks current generated files. I'll fix and rebase this PR afterwards.

Reply:
Fixed in upstream

Reply:
You are absolutely correct. For my paper type sticker_height = vertical_stride so I failed to notice the distinction.